app/models/query/processed_applications.rb

```diff
@@ -6,20 +6,20 @@ def initialize(user)
 
     def find
       @user.office.applications.
-        includes(:evidence_check, :payment).
-        references(:evidence_check, :payment).
+        includes(:evidence_check, :part_payment).
+        references(:evidence_check, :part_payment).
         where(where_condition).
         order(id: :asc)
     end
 
     def where_condition
-      <<WHERE
-(applications.application_outcome IS NOT NULL)
-AND
-(evidence_checks.id IS NULL OR evidence_checks.completed_at IS NOT NULL)
-AND
-(payments.id IS NULL OR payments.completed_at IS NOT NULL)
-WHERE
+      <<-WHERE.gsub(/^\s+\|/, '')
```

Comment:
Cool, didn't know that was a recommended style :+1: 

I didn't mean running tests, though, but the real app. This runs every time somebody access `/processed_application`. But yeah, not a biggie, I guess, I'm probably overthinking it.